lesson_11.md

```diff
@@ -11,7 +11,7 @@ Students will be able to...
 
 ## Materials/Preparation
 
--   [Unit 1 Tips][]
```

Comment:
This is an incorrect change. All links should be followed by a parenthesis or square-bracket expression. Some Markdown renderers forgive this, many do not (and instead render it as normal text with square brackets).